entity/FedexEntities.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entities xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/entity-definition-2.1.xsd">
+
+    <!--Default Authentication Entity for storing Credentials -->
+    <!--Entity to stored required credentials for Authentication-->
+    <entity entity-name="FedexTokenDetails" package="mantle.fedex">
```

Comment:
Please check the existing integration, and used the entity name as per the naming convenstion